src/cmd/tools/m3db_tool/main/common/http.go

```diff
@@ -0,0 +1,115 @@
+package common
```

Comment:
Please, avoid the use of package `common`. This is an antipattern as explained here https://blog.golang.org/package-names